app/application.py

```diff
@@ -0,0 +1,74 @@
+from typing import Any, Tuple
+from flask import render_template, request
+from assessment.app import create_app
+import json
+from assessment.models import db
+from assessment.models.customer import Customer
+from assessment.models.product import Product
+from typing import Tuple, Union, Dict
+
+app = create_app()
+
+def _standard_response(
+    payload:Union[str,dict], status_code:int
+) -> Tuple[str,int,Dict[str,str]]:
+
+    response = {'error': False}
+    if status_code != 200:
+        response['error'] = True
+        response['reason'] = payload
+    else:
+        response.update(payload)
+
+    return json.dumps(response), status_code, { 'Content-Type': 'application/json'}
+
+
+@app.route('/')
+def index():
+    # Default route - blank page
+    return render_template('index.html')
+
+
+@app.route('/order', methods=['POST'])
+def order_product():
```

Comment:
Should this file be purely about exposing app instance to be ran?
And so should routes like this live in `application/routes/orders` or something?

Or are we looking for feedback on this sort of thing too as part of this test?

Reply:
I have offered the candidate the opportunity to 'tidy' things up a bit. This is an obvious one to fix.